fast/stages-multitenant/0-bootstrap-tenant/outputs.tf

```diff
@@ -20,6 +20,15 @@ locals {
       "${path.module}/templates/workflow-${v.type}.yaml", (
         k == "bootstrap"
         ? {
+          allowed_audience = (
```

Comment:
this is unreadable :) can you explain what's the purpose of the new code blocks?

Reply:
I've only added the `allowed_audience` here, everything else is unchanged. I could've gone with `try()` instead of using ternary operator, but reading other PRs I've understood there's a preference of not using try() since it can hide some obvious errors. It basically gets the first value from the `allowed_audiences` property of the configured Workload Identity Federation pool.

Reply:
Our dislike of `try` is because it hides errors but also that it makes code harder to follow/read. I rather have the `try` if the alternative is 3 nested binaries (a "senary"?)

If you really want to avoid the `try`, consider making `allowed_audiences` an `optional(list(string), [])`

Reply:
Yeah, makes sense. Thanks for the suggestion, the new code should solve this